lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -32,11 +32,13 @@
             required: True
             env:
               - name: VMWARE_SERVER
+              - name: VMWARE_HOST
```

Comment:
Minor nit - if someone had both `VMWARE_SERVER`, and `VMWARE_HOST` set, VMWARE_SERVER was used. Now `VMWARE_HOST` has a higher precedence (last one wins). You could reorder to ensure compatibility:
```
            env:
              - name: VMWARE_HOST
              - name: VMWARE_SERVER
```
Probably unlikely to affect anyone, but perhaps worth mentioning.